lang/py/avro/constants.py

```diff
@@ -57,3 +56,8 @@
 )
 
 VALID_TYPES = PRIMITIVE_TYPES + NAMED_TYPES + ("array", "map", "union", "request", "error_union")
+
+INT_MIN_VALUE = -(1 << 31)
+INT_MAX_VALUE = (1 << 31) - 1
+LONG_MIN_VALUE = -(1 << 63)
+LONG_MAX_VALUE = (1 << 63) - 1
```

Comment:
I am fairly ambivalent about this change, because we don't appear to use these particular constants in more than one module. They may as well be "private". However, if it's useful externally for some reason, then I'm not against it.